modules/data-entry/src/main/frontend/src/vocabQuery/util.jsx

```diff
@@ -17,8 +17,8 @@
 //  under the License.
 //
 
-// Do not place an ending / in the URL below
-export const REST_URL = "https://services.phenotips.org/rest/vocabularies";
+// Ensure there is an endling / in the URL below
+export const REST_URL = window.location.origin + "/Vocabularies/";
```

Comment:
`window.location.origin` shouldn't be needed. Moreover, it might break things if, for example, we define a different [base url](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base) (we don't at this time, but let's keep our options open).

```suggestion
export const REST_URL = "/Vocabularies/";
```

Reply:
Turns out this won't actually work. We'll try to accomodate the mentioned usecase when we actually encounter it. The current code works just right for now.